prediction_market_agent/agents/microchain_agent/functions.py

```diff
@@ -80,7 +86,27 @@ def __call__(self) -> list[str]:
         ]
 
 
-class GetPropabilityForQuestion(MarketFunction):
+class GetMarketProbability(MarketFunction):
+    @property
+    def description(self) -> str:
+        return (
+            f"Use this function to get the probability of a 'Yes' outcome for "
+            f"a binary prediction market. This is equivalent to the price of "
+            f"the 'Yes' token in {self.currency}. Pass in the market id as a "
+            f"string."
+        )
+
+    @property
+    def example_args(self) -> list[str]:
+        return [get_example_market_id(self.market_type)]
+
+    def __call__(self, market_id: str) -> list[str]:
+        return [
+            str(self.market_type.market_class.get_binary_market(id=market_id).p_yes)
+        ]
+
+
+class PredictPropabilityForQuestion(MarketFunction):
```

Comment:
Renamed otherwise it could easily be mistaken for `GetMarketProbability`

Reply:
I'm a bit confused - shouldn't this method be named something like `PlaceBet` or `BuyTokensForOutcome`? It's not really predicting anything, like the description says it allows the agent to purchase tokens.

Reply:
@gabrielfior It's okay, just that Github hides the lines:

 
![Screenshot by Dropbox Capture](https://github.com/gnosis/prediction-market-agent/assets/15619339/3804a2e0-cf02-4bd5-89f9-d2a3640b4482)